src/pages/Home/Home.tsx

```diff
@@ -1,6 +1,9 @@
 import Layout from "../../components/Layout/Layout";
```

Comment:
I think you can remove this `Layout` component it is only in the project as an example and it is not relevant to this page